doc/source/release/1.17.0-notes.rst

```diff
@@ -29,84 +29,1046 @@ Highlights of this release
+  ``from_matrix`` is now also faster in cases where a known orthogonal matrix
+  is used.
+- The `scipy.spatial.geometric_slerp` function can now extrapolate. When given a
+  value outside the range [0, 1], ``geometric_slerp()`` will continue with
+  the same rotation outside this range. For example, if spherically
+  interpolating with ``start`` being a point on the equator, and ``end``
+  being a point at the north pole, then a value of ``t=-1`` would give you a
+  point at the south pole.
+- ``Rotation.as_euler`` and ``Rotation.as_davenport`` methods have gained a
+  ``suppress_warnings`` parameter to enable suppression of gimbal lock warnings.
 
 `scipy.special` improvements
 ============================
+- The following functions for statistical applications have significantly
+  improved parameter ranges and reduced error rates: ``btdtria``, ``btdtrib``,
+  ``chdtriv``, ``chndtr``, ``chndtrix``, ``chndtridf``, ``chndtrinc``, ``fdtr``,
+  ``fdtrc``, ``fdtri``, ``pdtrik``, ``stdtr`` and ``stdtrit``.
 
 
 `scipy.stats` improvements
 ==========================
+- `scipy.stats.matrix_t` has been added to represent the matrix t distribution.
+  It supports methods ``pdf`` (and ``logpdf``) for computing the probability
+  density function and ``rvs`` for generating random variates.
+- `scipy.stats.Logistic` was added for modeling random variables that follow a
+  logistic distribution.
+- `scipy.stats.quantile` now accepts a ``weights`` argument to specify
+  frequency weights.
+- `scipy.stats.quantile` is now faster on large arrays as it no longer uses
+  stable sort internally.
+- `scipy.stats.quantile` supports three new values of the ``method`` argument,
+  ``'round_inward'``, ``'round_outward'``, and ``'round_neareast'``, for use in
+  the context of trimming and winsorizing data.
+- `scipy.stats.truncpareto` now accepts negative values for the exponent shape
+  parameter, enabling use of ``truncpareto`` as a more general power law
+  distribution.
+- `scipy.stats.logser` now provides a distribution-specific implementation of
+  the ``sf`` method, improving speed and accuracy.
+- Implementations of the following function have been vectorized:
+  `scipy.stats.ansari`, `scipy.stats.cramervonmises`,
+  `scipy.stats.cramervonmises_2samp`, `scipy.stats.epps_singleton_2samp`,
+  `scipy.stats.fligner`, `scipy.stats.friedmanchisquare`, `scipy.stats.kruskal`,
+  `scipy.stats.ks_1samp`, `scipy.stats.levene`, and `scipy.stats.mood`.
+  Typically, this improves performance with multidimensional (batch) input.
+- The critical value tables of `scipy.stats.anderson` have been updated.
+- The speed and accuracy of most `scipy.stats.zipfian` methods has been
+  improved.
+- The accuracies of the `scipy.stats.Binomial` methods ``logcdf`` and
+  ``logccdf`` have been improved in the tails.
+- The default guess of ``scipy.stats.trapezoid.fit`` has been improved.
 
 
+**************************
+Array API Standard Support
```

Comment:
Maybe we should mention the new coverage table even though it isn't strictly a feature? https://scipy.github.io/devdocs/dev/api-dev/array_api.html#api-coverage